pkg/transforms/tags_test.go

```diff
@@ -56,14 +56,14 @@ func TestTags_AddTags(t *testing.T) {
 	tests := []struct {
 		Name          string
 		FunctionInput interface{}
-		TagsToAdd     map[string]string
+		TagsToAdd     map[string]interface{}
```

Comment:
This test can be removed since we have the one below

Reply:
test has been removed